web/templates/show_inactive_or_muted_channels.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="stream-list-toggle-inactive-or-muted-channels bottom_left_row zoom-in-hide">
```

Comment:
Code-wise, it'd be nice if we had a better name than `inactive-or-muted-channels` for this category -- a description around them being hidden behind the click would be nice. Don't have a great idea right now.